docs/docs/flags/output.1.md

```diff
@@ -11,7 +11,7 @@ tracee **\-\-output** - Control how and where output is printed
 
 ## SYNOPSIS
 
-tracee **\-\-output** <format[:file,...]\> | gotemplate=template[:file,...] | forward:url | webhook:url | option:{stack-addresses,exec-env,relative-time,exec-hash,parse-arguments,parse-arguments-fds,sort-events} ...
+tracee **\-\-output** <format[:file,...]\> | gotemplate=template[:file,...] | forward:url | webhook:url | option:{stack-addresses,exec-env,relative-time,exec-hash={pathname,dev-inode,digest-inode},parse-arguments,parse-arguments-fds,sort-events} ...
```

Comment:
pathname->inode

Reply:
if all of them are `XXX-inode`, should we remove `inode` ?